core/repository/account.ts

```diff
@@ -0,0 +1,44 @@
+import { Account, IAccountFilds } from 'shared/model/account';
+import Response from 'shared/model/response';
+
+class AccountRepo {
+    private memoryAccountsByAddress: { [address: number]: Account } = {};
```

Comment:
Here we have AccountRepo context and we not need to write about this context. For example, variable `memoryAccountsByAddress` should be `byAddress`. Method `addAccount` should be `add`, etc.